src/JWE.ts

```diff
@@ -1,10 +1,19 @@
 import { base64ToBytes, bytesToBase64url, decodeBase64url, toSealed } from './util'
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export type ProtectedHeader = Record<string, any>
+
+interface EphemeralPublicKey {
+  kty: string
+  crv: string
+  x: string
+}
```

Comment:
Yeah, but there is no RSA support now.
The only reason this is slightly more structured now is to avoid the use of `any` type.
When/if RSA will be supported, we'll also change this structure

Reply:
I added some optional params that would be usable for RSA